packages/rum-core/src/domain/resource/resourceUtils.ts

```diff
@@ -186,6 +186,10 @@ export function computeResourceEntryProtocol(entry: RumPerformanceResourceTiming
   return entry.nextHopProtocol === '' ? undefined : entry.nextHopProtocol
 }
 
+export function computeResourceContentType(entry: RumPerformanceResourceTiming) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Maybe a little comment to explain why we discard empty string.